crates/install-wheel-rs/src/linker.rs

```diff
@@ -37,6 +37,7 @@ pub struct Locks(Mutex<FxHashMap<PathBuf, Arc<Mutex<()>>>>);
 #[instrument(skip_all, fields(wheel = %filename))]
 pub fn install_wheel(
     layout: &Layout,
+    relocatable: bool,
```

Comment:
I moved these next to each other because part of me wishes that it were on `Layout` (though I don't think creating `layout` should require querying the filesystem, so leaving them separate).

Reply:
I did originally put this inside of `Layout`, but for some reason decided to back out of that.

> (though I don't think creating layout should require querying the filesystem, so leaving them separate).

Could use the `layout.as_relocatable()` pattern? (only mark it as relocatable if specifically requested by the caller)